sources/api/netdog/src/cli/install.rs

```diff
@@ -53,10 +53,13 @@ pub(crate) fn run(args: InstallArgs) -> Result<()> {
             InterfaceFamily::Ipv4 | InterfaceFamily::Ipv6,
         ) => {
             let lease = fetch_lease(&primary_interface, interface_type, args.data_file)?;
-            write_resolv_conf(&lease)?;
             write_current_ip(&lease.ip_address.addr())?;
+            write_primary_interface_sysctl(primary_interface)?;
 
-            write_primary_interface_sysctl(primary_interface)?
+            write_resolv_conf(&lease)?;
+            // Symlink resolv.conf to a common path, for compatibility reasons during
+            // systemd-networkd development
+            symlink_force(RESOLV_CONF, RESOLV_CONF_SYMLINK)?
```

Comment:
nit: to match the imperative naming of other functions called here:
```suggestion
            force_symlink(RESOLV_CONF, RESOLV_CONF_SYMLINK)?
```